### PR TITLE
ELEMENTS-320: Allows resolveEntry to be overriden and overrides it on…

### PR DESCRIPTION
**Pull request by pncosta**
… nuxeo-user-suggestion

### CODE REVIEW

File: widgets/nuxeo-select2-behavior.html [hunk 1]
```diff
@@ -131,6 +131,13 @@
         }
       },
 
+      resolveEntry: {
```
Comment (adilioaraujo) [inline widgets/nuxeo-select2-behavior.html]: Add docs description

File: widgets/nuxeo-user-suggestion.html [hunk 2]
```diff
@@ -136,6 +137,13 @@
       }
     },
 
+    resolveEntry: {
```
Comment (adilioaraujo) [inline widgets/nuxeo-user-suggestion.html]: Add docs description

File: widgets/nuxeo-select2-behavior.html [hunk 3]
```diff
@@ -131,6 +131,13 @@
...
+        type: Function,
+        value: function () {
```
Comment (Gabez0r) [inline widgets/nuxeo-select2-behavior.html]: no space between function and ()

File: widgets/nuxeo-user-suggestion.html [hunk 4]
```diff
@@ -136,6 +137,13 @@
...
+      type: Function,
+      value: function () {
```
Comment (Gabez0r) [inline widgets/nuxeo-user-suggestion.html]: no space between function and ()

**Comment by Gabez0r** [comment]
👍

**Comment by adilioaraujo** [comment]
:+1:

**Pull request merged**